ckanext/xloader/plugin.py

```diff
@@ -64,23 +67,54 @@ def configure(self, config_):
                     )
                 )
 
-    # IResourceUrlChange
+    # IDomainObjectModification
+
+    def notify(self, entity, operation):
+        # type: (ckan.model.Package|ckan.model.Resource, DomainObjectOperation) -> None
+        """
+        Runs before_commit to database for Packages and Resources.
+        We only want to check for changed Resources for this.
+        We want to check if values have changed, namely the url.
+        See: ckan/model/modification.py.DomainObjectModificationExtension
+        """
+        if operation != DomainObjectOperation.changed \
+        or not isinstance(entity, Resource):
+            return
+
+        # If the resource requires validation, stop here if validation
+        # has not been performed or did not succeed. The Validation
+        # extension will call resource_patch and this method should
+        # be called again. However, url_changed will not be in the entity
+        # once Validation does the patch.
+        if _is_validation_plugin_loaded() and \
+        toolkit.asbool(toolkit.config.get('ckanext.xloader.requires_validation')):
```

Comment:
Flake8 thinks this should be indented by 8 spaces.

Reply:
@ThrawnCA oops forgot flake8 hate the no indented wrapped if statements. I usually fix it in core with just two spaces after the if wrapping.

I have done that now, so lemme know if flake8 still hates it.